test/cypress/e2e/home.cy.ts

```diff
@@ -1,13 +1,17 @@
 const sizes: Cypress.ViewportPreset[] = ['iphone-6', 'macbook-13'];
 
```

Comment:
Just wondering about including a tablet viewport preset (e.g. '`ipad-2'`) preset given that BE displays the the search-form in three different configurations. Also expanding the desktop view to `'macbook-13'`

Reply:
Yeah, I can add that and re-try the test!